examples/colab/wav2vec2_saved_model_finetuning.ipynb

```diff
@@ -0,0 +1,2441 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    save_dir = "finetuned-wav2vec2"
</pre><p>It might be worth mentioning that they can set <code>include_optimizer=False</code>  if this is being exported only for inference. This keeps the model a bit smaller.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/hub/pull/788/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='73'/>